### PR TITLE
Add manifest for org.kde.skanlite

### PR DESCRIPTION
**Pull request by flyingcakes85**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned.   
    - `appdata.patch` : https://invent.kde.org/graphics/skanlite/-/merge_requests/29

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: org.kde.skanlite.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        "/lib/pkgconfig",
+        "/lib/*.a",
+        "/lib/*.la",
+        "/share/man"
+    ],
+    "finish-args": [
+        "--device=all",
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=x11"
```
Comment (hfiguiere) [inline org.kde.skanlite.json:18]:
```suggestion
        "--socket=fallback-x11"
```

File: org.kde.skanlite.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    ],
+    "modules": [
+        {
+            "name": "sane",
```
Comment (hfiguiere) [inline org.kde.skanlite.json:22]: don't you need libusb for that?

File: org.kde.skanlite.json [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/release-service/21.12.2/src/skanlite-21.12.2.tar.xz",
+                    "sha256": "15c181b3d155a7222279b52ca7dd83aed2fa02b2edda1cd3c211e8bbfdcf6dbb"
+                },
+                {
+                    "type": "file",
+                    "path": "logo.png",
```
Comment (hfiguiere) [inline org.kde.skanlite.json:54]: it could has well have the proper name from start.

Also, there in no upstream icon?

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: org.kde.skanlite.json (hunk 3 above)
Comment (flyingcakes85) [reply org.kde.skanlite.json]: Ok, it is provided by upstream too, but seems like we don't have 128x128 icon.

File: org.kde.skanlite.json (hunk 2 above)
Comment (flyingcakes85) [reply org.kde.skanlite.json:23]: Thanks, added.

**Comment by flyingcakes85** [comment]
bot, build org.kde.skanlite

**Comment by flyingcakes85** [comment]
@travier 

I don't have a scanner, so could not test. It does detect my webcam though.

File: org.kde.skanlite.json [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        "/lib/pkgconfig",
+        "/lib/*.a",
+        "/lib/*.la",
+        "/share/man"
+    ],
+    "finish-args": [
+        "--device=all",
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=x11"
```
Comment (hfiguiere) [inline org.kde.skanlite.json:18]:
```suggestion
        "--socket=fallback-x11"
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: org.kde.skanlite.json (hunk 4 above)
Comment (flyingcakes85) [reply org.kde.skanlite.json]: My bad. I remember making that change earlier, but it was not committed. 
Thanks for the patience.

**Comment by travier** [comment]
I should be able to find a scanner to give it a try.

**Comment by flyingcakes85** [comment]
bot, build org.kde.skanlite

**Comment by Erick555** [comment]
> I don't have a scanner, so could not test. It does detect my webcam though.

Who will maintain this app then? I think maintainer must be able to test if app works.

**Comment by flyingcakes85** [comment]
bot, build org.kde.skanlite

**Comment by barthalion** [comment]
@travier Please poke me when you manage to test it.

**Comment by flyingcakes85** [comment]
bot, build org.kde.skanlite

**Comment by travier** [comment]
> > I don't have a scanner, so could not test. It does detect my webcam though.
> 
> Who will maintain this app then? I think maintainer must be able to test if app works.

All KDE Apps are co-maintained bu the KDE Team. I have access to a scanner.

**Comment by flyingcakes85** [comment]
bot, build org.kde.skanlite

**Comment by flyingcakes85** [comment]
@travier 

The only difference I could spot was that the versions here were out of date. I have updated them.

Requesting you to please test again as per your convenience.

**Comment by travier** [comment]
OK, let's merge this one as Beta and we can improve it and test it as we go.

**Pull request closed without merging**